clickhouse/backup-clickhouse.sh

```
@@ -0,0 +1,162 @@
+    local backup_file="clickhouse-backup-${backup_date}.tar"
+    local temp_backup="/tmp/${backup_file}"
+
+    log "Starting backup..."
+    log "Source: $volume_path"
+    log "Destination: ${STORAGE_BOX_HOST}:${BACKUP_BASE_DIR}/${backup_file}"
+
+    # Create tar archive (no compression - ClickHouse data is already compressed)
+    # --ignore-failed-read: Continue even if files are deleted/changed during backup (normal for live DB)
+    log "Creating tar archive..."
+    if tar -cf "$temp_backup" --ignore-failed-read -C "$volume_path" . 2>&1 | tee -a "$LOG_FILE"; then
+        local archive_size=$(du -h "$temp_backup" | cut -f1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)